src/Raven.Studio/wwwroot/App/views/manage/cluster.html

```diff
@@ -27,6 +27,20 @@
                     </div>
                 </div>
             </div>
+            <div data-bind="visible: topology().isPassive()">
+                <div class="passive-node-info bg-info margin-top">
+                    <div class="padding">
+                        <p class="lead margin-bottom">The running server is in a <strong>Passive State</strong>, it is not part of a cluster yet.</p>
+                        <p>Either one of the following can be done to <strong>Bootstrap a Cluster</strong>:</p>
+                        <ul>
+                            <li>Create a new database</li>
+                            <li>Bootstrap the cluster (see button above)</li>
+                            <li>Add another node (see button above), resulting in both nodes being part of the cluster</li>
```

Comment:
another means there is already some node and we add second one?

Reply:
if you already checked that then no need to test that again.

Reply:
so theoretically, it should work,
but when I try to add another (passive) node to the current passive node
I am always required to register a license first... 
which is an action that by itself causes the first node to become a cluster..
so adding the node is not a relevant action anymore

Reply:
As discussed w/ @grisha-kotler 
yes - adding another node will result in both of them being in a cluster
so - leaving this text as-is.